src/osd/pg_scrubber.cc

```diff
@@ -1889,7 +1901,11 @@ PgScrubber::PgScrubber(PG* pg)
     , m_pg_whoami{pg->pg_whoami}
     , preemption_data{pg}
 {
-  m_fsm = std::make_unique<ScrubMachine>(m_pg, this);
+  std::stringstream prefix;
+  prefix << "osd." << m_osds->whoami << " scrubber pg(" << m_pg_id << "): ";
```

Comment:
could you use pg[ rather than pg( for all of these prefixes, so that it'll match existing greps for pg-level logging?